### ISSUE TITLE
No naga devices found or you don't have permission to access them.

### ISSUE THREAD

**Issue by morgandavidson42** [apocatarsis/Naga_KeypadMapper#78]
using naga trinity on pop_os 22.04
I verified that naga.cpp has the device
probably related, when I use xev to test the side numbers, they don't register
any advice?

**Comment by morngrar** [comment]
I have the same issue on arch, but the side buttons register as number keys in xev

**Comment by morngrar** [comment]
The solution is to reboot, ref. #61

**Comment by lostallmymoney** [comment]
Yes reboot, for more functionnalities you can try https://github.com/lostallmymoney/Razer_Mouse_Linux where you might also have to reboot. ;)
